recipes/protobufc/bld.bat

```diff
@@ -0,0 +1,16 @@
+
+cd build-cmake
+mkdir build
+cd build
+
+cmake -G "Ninja" ^
+      %CMAKE_ARGS% ^
+      -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+      -DCMAKE_BUILD_TYPE:STRING=Release ^
+      -DCMAKE_LIBRARY_PATH="%LIBRARY_LIB%" ^
+      -DCMAKE_INCLUDE_PATH="%INCLUDE_INC%" ^
```

Comment:
```suggestion
```

Reply:
updated as suggested in 53920f3b508d2968e2db5c0c1e80f8bd41575a76